app/action/map-track-actions.coffee

```diff
@@ -0,0 +1,13 @@
+config     = require '../config'
+
+startMapTrack = (actionContext, payload, done) ->
```

Comment:
Should we name "MapTrack" rather "MapTracking". It would make more sense for me.